receiver/otlpreceiver/README.md

```diff
@@ -1,14 +1,42 @@
 # OpenTelemetry Receiver
 
-This is the default receiver for the OpenTelemetry project.
+Receives traces and/or metrics via gRPC using
+[OpenTelemetry](https://opentelemetry.io/) format.
 
 To get started, all that is required to enable the OpenTelemetry receiver is to
-include it in the receiver definitions. This will enable the default values as
-specified [here](./factory.go).
-The following is an example:
+include it in the receiver definitions.
+
+The following settings are required:
+
+- `endpoint` (default = 0.0.0.0:55680): host:port to which the exporter is
+  going to receive traces or metrics, using the gRPC protocol. The valid syntax
+  is described at https://github.com/grpc/grpc/blob/master/doc/naming.md.
+- `transport` (default = tcp): which transport to use between `tcp` and `unix`.
+
+The following settings are optional:
+
+- `cors_allowed_origins` (default = unset): allowed CORS origins for HTTP/JSON
+  requests. See the HTTP/JSON section below.
+- `keepalive`: see
+  https://godoc.org/google.golang.org/grpc/keepalive#ServerParameters for more
+  information
+  - `MaxConnectionIdle` (default = infinity)
+  - `MaxConnectionAge` (default = infinity)
+  - `MaxConnectionAgeGrace` (default = infinity)
+  - `Time` (default = 2h)
+  - `Timeout` (default = 20s)
+- `max_recv_msg_size_mib` (default = infinity): sets the maximum size of messages accepted
+- `max_concurrent_streams`: sets the limit on the number of concurrent streams
+- `tls_credentials` (default = unset): configured the receiver to use TLS. See
+  TLS section below.
+
+Examples:
+
 ```yaml
 receivers:
   otlp:
+  otlp/withendpoint:
+    endpoint: 127.0.0.1:55680
```

Comment:
Use `0.0.0.0:55680`?

Reply:
`0.0.0.0:55680` is the default configuration -- this shows how to change it to something non-default.